src/main/java/com/conveyal/datatools/manager/models/transform/NormalizeFieldTransformation.java

```diff
@@ -0,0 +1,354 @@
+package com.conveyal.datatools.manager.models.transform;
+
+import com.conveyal.datatools.common.status.MonitorableJob;
+import com.conveyal.datatools.manager.models.TableTransformResult;
+import com.conveyal.gtfs.loader.Field;
+import com.conveyal.gtfs.loader.Table;
+import com.csvreader.CsvReader;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.text.WordUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.supercsv.io.CsvListWriter;
+import org.supercsv.prefs.CsvPreference;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.io.StringWriter;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+import java.util.zip.ZipFile;
+
+import static com.conveyal.datatools.manager.DataManager.getConfigPropertyAsText;
+import static com.conveyal.gtfs.loader.Field.getFieldIndex;
+
+public class NormalizeFieldTransformation extends ZipTransformation {
```

Comment:
Please add some JavaDoc for this class.

Reply:
Thanks, updated in https://github.com/ibi-group/datatools-server/pull/371/commits/cd883d91e7d11fa2c698675980d8443f6869943f.